pyproject.toml

```diff
@@ -102,6 +102,7 @@ types-filelock = "^3.2.5"
 types-Markdown = "^3.3.21"
 types-futures = "^3.3.8"
 paramiko = "^2.11.0"
+slack-sdk = "^3.17.0"
```

Comment:
please remove from here, as it's used in dev-dependencies, and re-run `poetry lock`

Reply:
yeah apparently poetry added that to both general and dev-dependancies even though i specifically gave the --dev flag, weird.   ```poetry add slack-sdk --dev```